ui/apps/platform/src/Containers/AccessControl/AccessControlNoPermission.tsx

```diff
@@ -2,15 +2,26 @@ import React, { ReactElement } from 'react';
 import { Alert, AlertVariant, PageSection, PageSectionVariants } from '@patternfly/react-core';
 
 import AccessControlHeading from './AccessControlHeading';
+import { AccessControlEntityType } from '../../constants/entityTypes';
 
-function AccessControlNoPermission(): ReactElement {
+type AccessControlNoPermissionProps = {
+    subPage: string;
+    entityType?: AccessControlEntityType;
+    isNavHidden?: boolean;
+};
+
+function AccessControlNoPermission({
+    subPage,
+    entityType,
+    isNavHidden = false,
+}: AccessControlNoPermissionProps): ReactElement {
     return (
         <>
-            <AccessControlHeading isNavHidden />
+            <AccessControlHeading isNavHidden={isNavHidden} entityType={entityType} />
             <PageSection variant={PageSectionVariants.light}>
                 <Alert
                     className="pf-u-mt-md"
-                    title="You do not have permission to view Access Control"
+                    title={`You do not have permission to view ${subPage}`}
```

Comment:
This is okay for current requirements. When we move responsibility from container to main body routing, it renders generic 404 page which for purpose of security does not reveal whether a page exists.